package.json

```diff
@@ -1,14 +1,14 @@
 {
-  "name": "atom-package-manager",
+  "name": "@atom-ide-community/atom-package-manager",
   "description": "Atom package manager",
```

Comment:
"Community build of the Atom package manager"?

Reply:
Yes, that's a good idea.